src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTabIndex = tabs.findIndex(tab => tab.id === activeTabId);
+  const isTabActive = activeTabIndex !== -1;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab, index) => (
+            <li
+              className={cn({
+                'is-active':
+                  (isTabActive && activeTabIndex === index) ||
+                  (!isTabActive && index === 0),
+              })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => tab.id !== activeTabId && onTabSelected(tab.id)}
```

Comment:
Move this logic to the helper function and use it here

Reply:
ok